tinycc/distutils.py

```diff
@@ -41,22 +41,32 @@ def get_msvcr():
     """
     msc_pos = sys.version.find('MSC v.')
     if msc_pos != -1:
-        msc_ver = sys.version[msc_pos+6:msc_pos+10]
```

Comment:
```
import platform
platform.python_compiler()
```

is a much easier way of finding the compiler version

Reply:
fixed (hello, by the way, hope you guys are well!)

Reply:
Thanks @tcbennun!  Missed you at code camp in Poland a few months back 😄